antora-ui-camel/src/js/vendor/alogliasearch.bundle.js

```diff
@@ -93,5 +96,11 @@
         results.innerHTML = ''
       }
     }), 150)
+    cancel.addEventListener('click', function () {
+      container.className = 'navbar-search results-hidden'
+      results.innerHTML = ''
+      search.value = ''
+      cancel.style.display = 'none'
```

Comment:
Nitpicking: this and the block from 30-32 could be pulled in a separate function.